packages/html/cspell-ext.json

```diff
@@ -28,7 +28,22 @@
             // Adding patterns to the "includeRegExpList" to only include matching patterns
             "includeRegExpList": [],
             // To exclude patterns, add them to "ignoreRegExpList"
-            "ignoreRegExpList": [],
+            "ignoreRegExpList": [
+                "href",
```

Comment:
Wasn't sure if I should use this on as a pre-defined regex in the other library or just have it's own

Reply:
Please add them as named patterns.
I know it seems like a pain, but it allows someone else to override them.

Reply:
Like this:
```js
    // Define HTML related patterns
    // cspell:ignore IDREF activedescendant describedby errormessage flowto labelledby
    "patterns": [
        { "name": "HTML-IDREF-id",                       "pattern": "id=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-src",                      "pattern": "src=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-class",                    "pattern": "class=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-activedescendant",    "pattern": "aria-activedescendant=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-controls",            "pattern": "aria-controls=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-describedby",         "pattern": "aria-describedby=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-details",             "pattern": "aria-details=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-errormessage",        "pattern": "aria-errormessage=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-flowto",              "pattern": "aria-flowto=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-labelledby",          "pattern": "aria-labelledby=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-aria-owns",                "pattern": "aria-owns=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-for",                      "pattern": "for=\"(?:[^\\\"]+|\\.)*\"" },
        { "name": "HTML-IDREF-data",                     "pattern": "data-(.*[=> ]?)" }
    ],
    // Language Rules to apply to matching files.
    // Files are matched on `languageId` and `local`
    "languageSettings": [
        {
            "languageId": "html",
            "ignoreRegExpList": [
                "href",
                "HTML-IDREF-id",
                "HTML-IDREF-src",
                "HTML-IDREF-class",
                "HTML-IDREF-aria-activedescendant",
                "HTML-IDREF-aria-controls",
                "HTML-IDREF-aria-describedby",
                "HTML-IDREF-aria-details",
                "HTML-IDREF-aria-errormessage",
                "HTML-IDREF-aria-flowto",
                "HTML-IDREF-aria-labelledby",
                "HTML-IDREF-aria-owns",
                "HTML-IDREF-for",
                "HTML-IDREF-data"
            ],
            "dictionaries": ["html"]
        }

```